### DISCUSSION TITLE
Install apache-airflow problem

### DISCUSSION THREAD

**Discussion by italollima** [apache/airflow#26996] [Q&A]
![image](https://user-images.githubusercontent.com/68879590/195178555-6809e592-ba50-4e03-8531-8a9859946019.png)

hello,
How can i solve this errors ?

**Comment by potiuk** [comment]
You completely forgot to mention which versions of Airflow and openssl you have. I suggest to upgrade both to latest versions.

This is pretty expected that when you upgrade OS, some older version of libraries and apps expose some security related problems and expect ypu to use latest versions of the software you are installing. I suggest you follow that pattern.
